src/main/features/offer/form/models/defendantResponse.ts

```diff
@@ -0,0 +1,30 @@
+import { IsDefined, IsIn } from 'class-validator'
+import { Serializable } from 'models/serializable'
+
+export class ValidationErrors {
+  static readonly OPTION_REQUIRED: string = 'Choose option: yes or no or make an offer'
+}
+
+export class DefendantResponse implements Serializable<DefendantResponse> {
+  @IsDefined({ message: ValidationErrors.OPTION_REQUIRED })
+  @IsIn(['yes', 'no', 'makeAnCounterOffer'], { message: ValidationErrors.OPTION_REQUIRED })
```

Comment:
These values should be defined in a constants class , e.g. like in [`EvidenceType`](https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/features/response/form/models/evidenceType.ts). This constants class should be also exported as a nunjucks global, see [here](https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/modules/nunjucks/index.ts#L78) for details.